src/Components/Header/index.js

```diff
@@ -24,17 +24,19 @@ function Header({ selected, setSelected }) {
         style={styles.logoContainer}
         underlayColor="transparent"
         onPress={() => setSelected(0)}
+        accessible={true}
+        accessibilityLabel={titles[0]}
       >
         <Image
           style={{ height: 50, width: 100 }}
           source={require('./../../assets/logo.png')}
         />
       </TouchableHighlight>
-      {MenuItem(1, selected, setSelected, 'About us')}
-      {MenuItem(2, selected, setSelected, 'Programs')}
-      {MenuItem(3, selected, setSelected, 'Projects')}
-      {MenuItem(4, selected, setSelected, 'Events')}
-      {MenuItem(5, selected, setSelected, 'Contribute')}
+      {MenuItem(1, selected, setSelected, titles[1])}
+      {MenuItem(2, selected, setSelected, titles[2])}
+      {MenuItem(3, selected, setSelected, titles[3])}
+      {MenuItem(4, selected, setSelected, titles[4])}
+      {MenuItem(5, selected, setSelected, titles[5])}
```

Comment:
I thought of adding `{MenuItem(0, selected, setSelected, titles[0])}` since HOME is missing.

Reply:
@gaurivn The AnitaB.org logo acts as the home button

Reply:
Ah okay I just checked now, all good. Thanks @sidvenu for pointing it out.